Source/Hurl.BrowserSelector/Windows/MainWindow.xaml

```diff
@@ -2,174 +2,141 @@
+        </Grid.RowDefinitions>
 
-                <StackPanel
-                    HorizontalAlignment="Right"
-                    VerticalAlignment="Center"
-                    Orientation="Horizontal">
-                    <wpfui:DropDownButton
-                        Margin="0,0,4,0"
-                        HorizontalAlignment="Right"
-                        VerticalAlignment="Center"
-                        Background="#20000000"
-                        FontSize="16"
-                        Icon="{wpfui:SymbolIcon Navigation24}">
-                        <wpfui:DropDownButton.Flyout>
-                            <ContextMenu>
-                                <wpfui:MenuItem
-                                    Click="Button_Click_1"
-                                    Header="Rules"
-                                    Tag="settings">
-                                    <wpfui:MenuItem.Icon>
-                                        <wpfui:SymbolIcon Symbol="Add24" />
-                                    </wpfui:MenuItem.Icon>
-                                </wpfui:MenuItem>
-                                <wpfui:MenuItem
-                                    Click="Button_Click"
-                                    Header="Timed"
-                                    Tag="reload">
-                                    <wpfui:MenuItem.Icon>
-                                        <wpfui:SymbolIcon Symbol="Clock24" />
-                                    </wpfui:MenuItem.Icon>
-                                </wpfui:MenuItem>
-                                <wpfui:MenuItem
-                                    Click="SettingsBtnClick"
-                                    Header="Settings"
-                                    Tag="exit">
-                                    <wpfui:MenuItem.Icon>
-                                        <wpfui:SymbolIcon Symbol="Settings24" />
-                                    </wpfui:MenuItem.Icon>
-                                </wpfui:MenuItem>
-                            </ContextMenu>
-                        </wpfui:DropDownButton.Flyout>
-                    </wpfui:DropDownButton>
-                    <wpfui:Button
-                        Margin="0,0,4,0"
-                        Padding="7,5"
-                        Background="Transparent"
-                        BorderBrush="Transparent"
-                        Click="CloseBtnClick"
-                        Cursor="Hand"
-                        FontSize="16"
-                        ToolTip="Minimize to Tray">
-                        <wpfui:Button.Icon>
-                            <wpfui:SymbolIcon Symbol="Dismiss24" />
-                        </wpfui:Button.Icon>
-                    </wpfui:Button>
-                </StackPanel>
-            </Grid>
+        <wpfui:TitleBar
+            Title="Select Browser"
+            Height="32"
+            CloseClicked="HideMainWindow"
+            FontSize="14"
```

Comment:
```suggestion
            Title="Select Browser"
            CloseClicked="HideMainWindow"
```
Height and FontSize don't need to be hardcoded. There's a possibility the size isn't even implemented: https://github.com/lepoco/wpfui/issues/937